ignite/pkg/dirchange/dirchange.go

```diff
@@ -6,47 +6,40 @@ import (
 	"errors"
 	"os"
 	"path/filepath"
+
+	"github.com/ignite-hq/cli/ignite/pkg/cache"
 )
 
 var ErrNoFile = errors.New("no file in specified paths")
 
-// SaveDirChecksum saves the md5 checksum of the provided paths (directories or files) in the specified directory
+// SaveDirChecksum saves the md5 checksum of the provided paths (directories or files) in the provided cache
 // If checksumSavePath directory doesn't exist, it is created
 // paths are relative to workdir, if workdir is empty string paths are absolute
-func SaveDirChecksum(workdir string, paths []string, checksumSavePath string, checksumName string) error {
```

Comment:
To consolidate caching strategies I rewrote this to use the cache as storage instead. 
Another approach I considered was to merge this functionality directly into the cache package so we could do something like `myCache.HasDirChanged(workdir, paths)` which could be prettier.